whi_imu:
  loop_hz: 50 # hz
  frame_id: imu
  data_topic: imu_data
  mag_topic: mag_data
  temp_topic: temp_data
  hardware_interface:
    module: JY61
    port: /dev/imu
    baudrate: 115200
    pack_length: 11
    # JY-61 is 0xff 0xaa 0x52
    reset_yaw: [0xff, 0xaa, 0x52]
    instruction_min_span: 5 # unit millisecond
    with_magnetic: true
    with_temperature: false
